recipes/opennavsurf-bag/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set version = "2.0.1" %}
+
+package:
+  name: opennavsurf-bag
+  version: {{ version }}
+
+source:
+  url: https://github.com/OpenNavigationSurface/BAG/archive/refs/tags/release-{{ version }}.tar.gz
+  sha256: 7b246eec04f46782b74bc9bf14be5ddf0a157360d17bd2f48a150992c540025d
+
+build:
+  number: 0
+  skip: true  # [py<38]
+  skip_compile_pyc:
+    python/*.py
+
+requirements:
+  build:
+    - cmake
+    - ninja
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - sysroot_linux-64  # [linux64]
+    - python  # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - pip
```

Comment:
Pleasure remove all the (pure) Python dependencies from build. They should be in host.